src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/TraceCollection/TraceCollector.cs

```diff
@@ -40,7 +39,14 @@ public sealed class TraceCollector : IDisposable
         {
             { CollectType.gc, "gcCollectOnly" },
             { CollectType.cpu,  "collect_cpu" },
-            { CollectType.threadtime, "collect_threadTime" },
+            { CollectType.threadtime, "collect_threadTime" }
+        };
+
+        internal static readonly Dictionary<CollectType, string> LinuxLocalRunCollectTypeMap = new()
+        {
+            { CollectType.gc, "--clrevents gc" },
+            { CollectType.cpu,  "--clrevents gc+stack --clreventlevel informational" },
+            { CollectType.threadtime, "--clrevents gc --clreventlevel Verbose" },
```

Comment:
This configuration doesn't match with the threadtime based trace collection - let's get rid of this until we find a good enough alternative.